osmaxx/excerptexport/models/export.py

```diff
@@ -27,11 +39,16 @@ class Export(models.Model):
     - the transformation of the data from the data sources' schemata (e.g. ``osm2pgsql`` schema) to the OSMaxx schema
     - the actual export to one specific GIS or navigation file format with one specific set of parameters
     """
+    INITIAL = 'initial'
+    INITIAL_CHOICE = (INITIAL, _('initial'))
+    STATUS_CHOICES = (INITIAL_CHOICE,) + statuses.STATUS_CHOICES
+
     extraction_order = models.ForeignKey('excerptexport.ExtractionOrder', related_name='exports',
                                          verbose_name=_('extraction order'))
     file_format = models.CharField(choices=FORMAT_CHOICES, verbose_name=_('file format / data format'), max_length=10)
     conversion_service_job_id = models.IntegerField(verbose_name=_('conversion service job ID'), null=True)
     status = models.CharField(_('job status'), choices=STATUS_CHOICES, default=INITIAL, max_length=20)
+    finished = models.DateTimeField(_('finished on'), default=None, blank=True, editable=False, null=True)
```

Comment:
As these is a datetime, not a bare date, use "at" rather than "on" for the display name. You might want to rename the field itselve to `finished_at`, too, as the current name seems to imply a boolean type.